lib/ii.c

```diff
@@ -12962,14 +13066,19 @@ grn_ii_builder_register_chunks(grn_ctx *ctx, grn_ii_builder *builder)
 
   a = array_get(ctx, builder->ii, builder->chunk.tid);
   if (!a) {
+    grn_obj token;
+    grn_ii_get_token_from_token_id(ctx, builder->ii, builder->chunk.tid, &token);
```

Comment:
It'll cause "variable definitions after expression".
We should call it after ` DEFINE_NAME`.